ffmpeg-2404-sdk/snapcraft.yaml

```diff
@@ -480,5 +480,5 @@ parts:
     after: [deps]
     override-prime: |
       craftctl default
-      sed -i '/usr\/lib\/x86_64-linux-gnu\/libvulkan.so.1\*/d' ${CRAFT_PART_SRC}/lists/*.list
+      sed -i "\\|usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libvulkan.so.1\\*|d" ${CRAFT_PART_SRC}/lists/*.${CRAFT_ARCH_BUILD_FOR}.list 
```

Comment:
> `CH_BUILD_FOR}.list `

A wild trailing space appears!

Reply:
Hii I have done the changes, also pushed another small change in the CI which allowed rc tags. Please check this out.